docs/admin/setup.rst

```diff
@@ -68,3 +68,31 @@ method with the ``trust`` method:
 .. code-block:: bash
 
     $ edgedb --admin -h </data/dir> configure insert auth --method=trust
+
+
+Using remote PostgreSQL cluster as a backend
```

Comment:
Please, document environment and pgpass file

Reply:
Done.